exercises/B-hello-world/exercise.js

```diff
@@ -1 +1,3 @@
+//noconsole.log("Hello world");
```

Comment:
Hi Shima, we don't usually leave commented out code in our pull requests. 

You can read more about it here :) https://syllabus.codeyourfuture.io/guides/code-style-guide#dont-leave-lots-of-commented-out-code